packages/grafana-data/src/themes/createShape.ts

```diff
@@ -11,6 +11,10 @@ export interface ThemeShapeInput {
 export function createShape(options: ThemeShapeInput): ThemeShape {
   const baseBorderRadius = options.borderRadius ?? 2;
 
+  /**
+   * @deprecated Use default values instead
```

Comment:
I was thinking we explicitly add new tokens like
```
    radius: {
      default: '2px',
      pill: '9999px',
      circle: '100%',
    },
```

and then here recommend using those instead and replace where we use `theme.shape.borderRadius()` with `theme.shape.radius.default`